content/en/teams.md

```diff
@@ -0,0 +1,369 @@
+---
+title: Teams
+sidebar: false
+---
+
+{{< team org="scipy" name="Maintainers" >}}
```

Comment:
I suggest using the script to put each team into a separate file, and to then use an include shortcode to pull them in.  Since we don't have an include shortcode yet, I will add one to the theme now.